doc/integrator/ogc_api.rst

```diff
@@ -0,0 +1,53 @@
+.. _integrator_ogc_api:
+
+OGC API Features
+----------------
+
+The OGC API is a new standard that defines modular API following REST standard to access spatial data.
+The current implementation is only on the server side and is based on the OGC API - Features standard and only works with the `items` request using a bbox filter.
```

Comment:
The current implementation of what? 
I assume we are describing what parts of the OCG API GeoMapFish implements?